src/test/java/github/weichware10/util/db/PermissionsTest.java

```diff
@@ -0,0 +1,71 @@
+package github.weichware10.util.db;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import io.github.cdimascio.dotenv.Dotenv;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'io.github.cdimascio.dotenv.Dotenv' import. Should be before 'org.junit.Test'.